docs/diga_api.md

```diff
@@ -0,0 +1,289 @@
+# DiGA API documentation
+
+This document aims to help DiGA manufacturers figure out how prescription codes can be verified and used for billing.
```

Comment:
After going through the document I think I would like it to be clarified at the top what this document is for and what it is not for. Something like:

```suggestion
This documents contains information for DiGA manufacturers on how the DiGA API works regarding prescription code validation and billing. It is targeted at manufacturers who wish to develop their own integration with the DiGA API or are curious to know how it works. If you are looking for an out-of-the-box solution that already works, you can use the [diga api client](https://github.com/alex-therapeutics/diga-api-client) provided in this repository as a java library, or you can use the [diga api service](https://github.com/gtuk/diga-api-service) (work in progress), which is an api wrapper around this library, as a microservice in your stack. Information on how to use these solutions is provided in their readmes and code documentation, and not in this document.
```

just to make sure you dont end up reading this thinking you will find out how to use the library (you should read the README for that) or just looking to use something that works already

Reply:
yes, makes sense! :)